### PR TITLE
change deprecated pkg_resources to importlib

### PR DESCRIPTION
**Pull request by fgrunewald**
Remove deprecated pkg_resources. For more details see [here](https://github.com/marrink-lab/vermouth-martinize/pull/546)

### CODE REVIEW

**Comment by fgrunewald** [comment]
@pckroon not sure why the tests fail here. Locally for me the paths are resolved correctly

File: polyply/__init__.py [hunk 1]
```diff
@@ -23,10 +23,10 @@
     import atexit
     from contextlib import ExitStack
 except ImportError:
-    import os
-    DATA_PATH = os.path.join(os.path.dirname(__file__), 'data')
-    TEST_DATA = os.path.join(os.path.dirname(__file__), 'tests/test_data')
-    del os
+    from pathlib import Path
+    DATA_PATH = Path(__file__).parent / 'data'
+    TEST_DATA = Path(__file__).parent / 'tests/test_data'
```
Comment (pckroon) [inline polyply/__init__.py:28]:
```suggestion
    TEST_DATA = Path(__file__).parent / 'tests' / 'test_data'
```

Comment (pckroon) [reply polyply/__init__.py:28]: Also in other places. Otherwise you lose the OS independence again.

File: polyply/tests/test_gen_params.py [hunk 2]
```diff
@@ -25,99 +25,76 @@
...
-        ff_group.add_argument('-lib', dest='lib', required=False, type=str,
-                              help='force-fields to include from library', nargs='*')
-        dna_group = parser.add_argument_group('DNA specifc options')
-        dna_group.add_argument('-dsdna', dest='dsdna', action='store_true',
-                               help='complement single sequence to dsDNA sequence')
-
-        args = parser.parse_args(args_in)
-        gen_params(**vars(args))
+@pytest.mark.parametrize('inpath, seq, seqf, name, ref_file', (
+    ([TEST_DATA / "gen_params/input/PEO.martini.3.itp"],
```
Comment (pckroon) [inline polyply/tests/test_gen_params.py:29]: For example here

**Review by pckroon — CHANGES_REQUESTED** [review]

File: polyply/__init__.py (hunk 1 above)
Comment (fgrunewald) [reply polyply/__init__.py:28]: I changed all I found but might have missed some. That's fine though. Having the tests OS-independent is good to have but not 100% essential. We can catch the rest at a later stage.

**Review by pckroon — APPROVED** [review]
LGTM :)

**Pull request merged**